students/AnastasiaPletser/hw-4/answer_1.js

```diff
@@ -0,0 +1,13 @@
+let vowels = ["a", "e", "o", "y", "u"];
+
+function searchVowels (initialText) {
+    let count = 0;
+    
+    for(let i = 0; i < vowels.length; i++)   {
+        if(vowels.includes(initialText[i])) {
+            count++
+        }
+    }
+    return console.log (count)
```

Comment:
this will always return `undefined`. 
```suggestion
    console.log(count);
    return count;
```